urxvt incompatiblility

Unable to run `bq` tool in urxvt terminal due to:
```
Error running command 'config': Command exception: 'java.lang.StringIndexOutOfBoundsException: String index out of range: 5'.
io.bootique.di.DIRuntimeException: Error invoking provider method 'createLineReader()' of module 'io.bootique.tools.shell.module.BQShellModule'

 Injection trace: 

 [0] resolving key <BindingKey: org.jline.reader.LineReader>
    -> Invoking provider method 'createLineReader()' of module 'io.bootique.tools.shell.module.BQShellModule'

Caused by: java.io.IOError: java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at org.jline.utils.Curses.tputs(Curses.java:62)
	at org.jline.utils.Curses.tputs(Curses.java:45)
	at org.jline.keymap.KeyMap.key(KeyMap.java:243)
	at org.jline.reader.impl.LineReaderImpl.key(LineReaderImpl.java:5784)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at org.jline.reader.impl.LineReaderImpl.bindKeys(LineReaderImpl.java:5792)
	at org.jline.reader.impl.LineReaderImpl.emacs(LineReaderImpl.java:5503)
	at org.jline.reader.impl.LineReaderImpl.defaultKeyMaps(LineReaderImpl.java:5479)
	at org.jline.reader.impl.LineReaderImpl.<init>(LineReaderImpl.java:277)
	at org.jline.reader.LineReaderBuilder.build(LineReaderBuilder.java:115)
	at io.bootique.tools.shell.module.BQShellModule.createLineReader(BQShellModule.java:182)
	at java.lang.reflect.Method.invoke(Method.java:566)
	at io.bootique.di.spi.ProvidesHandler$ProvidesMethodProvider.get(ProvidesHandler.java:180)
	at io.bootique.di.spi.TraceableProvider.get(TraceableProvider.java:49)
	at io.bootique.di.spi.DefaultScopeProvider.get(DefaultScopeProvider.java:50)
	at io.bootique.tools.shell.JlineShell.readCommand(JlineShell.java:124)
	at io.bootique.tools.shell.command.StartShellCommand.commandLoop(StartShellCommand.java:79)
	at io.bootique.tools.shell.command.StartShellCommand.run(StartShellCommand.java:66)
	at io.bootique.run.DefaultRunner.run(DefaultRunner.java:43)
	at io.bootique.BQRuntime.run(BQRuntime.java:81)
	at io.bootique.Bootique.exec(Bootique.java:335)
	at io.bootique.tools.shell.Application.main(Application.java:47)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.StringLatin1.charAt(StringLatin1.java:47)
	at java.lang.String.charAt(String.java:693)
	at org.jline.utils.Curses.doTputs(Curses.java:351)
	at org.jline.utils.Curses.tputs(Curses.java:60)
	... 29 more
```
This happens, due to outdated Jline included in build. (see https://github.com/jline/jline3/issues/533) After 3.15.0 works fine, also tested with latest 3.21.0.

--- a/pom.xml
+++ b/pom.xml
@@ -37,7 +37,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <bootique.version>2.0.RC1</bootique.version>
-        <jline.version>3.11.0</jline.version>
+        <jline.version>3.21.0</jline.version>
         <mustache.java.version>0.9.6</mustache.java.version>
         <native.image.plugin.version>21.2.0</native.image.plugin.version>
         <junit.version>4.13.2</junit.version>

